.travis.yml

```diff
@@ -1,22 +1,35 @@
 language: php
 
-dist: trusty
+dist: xenial
+
+services:
+  - mysql
+  - postgresql
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+addons:
+  apt:
+    packages:
+      - tidy
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=2.x-dev
+    - COMPOSER_ROOT_VERSION=2.0.x-dev
 
 matrix:
   include:
     - php: 5.6
-      env: DB=MYSQL RECIPE_VERSION=4.2.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
-    - php: 5.6
-      env: DB=MYSQL RECIPE_VERSION=4.3.x-dev PHPUNIT_TEST=1
-    - php: 7.0
-      env: DB=PGSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_TEST=1 PHPCS_TEST=1
     - php: 7.1
-      env: DB=PGSQL RECIPE_VERSION=4.4.x-dev PHPUNIT_COVERAGE_TEST=1
+      env: DB=MYSQL RECIPE_VERSION=4.5.x-dev PHPUNIT_TEST=1 PHPUNIT_COVERAGE_TEST=1
```

Comment:
There's no need to run the tests twice, you can remove `PHPUNIT_TEST=1` here since `PHPUNIT_COVERAGE_TEST=1` does it already (with coverage enabled)

Reply:
ok will do, did not realise that it did it already